requirements.txt

```diff
@@ -1,18 +1,17 @@
-# TODO: Update with exact module version
-numpy
-torch>=1.7
-opencv_python
+numpy==1.19.*
+torch==1.4.*
+opencv_python==4.5.1.48
+scipy==1.5.4
+matplotlib==2.1.1
```

Comment:
I think we need to loosen these if we want this to work well with other packages. This is very strict.

Reply:
To have cloud compatibility, we'll need to loosed these up.

Reply:
I think maybe we just loosen them without too much testing and next time we need to install this, we can do a more serious exercise of testing which versions are hard limits and what isn't.